xoto3/dynamodb/write_versioned/ddb_api.py

```diff
@@ -257,5 +261,7 @@ def item_remains_unmodified(
             ]
         )
 
-    args["TransactItems"] = transact_items
+    args: Dict[str, Any] = dict(TransactItems=transact_items)
+    if ClientRequestToken:
```

Comment:
Just curious. What is this doing for us?

Reply:
nothing right now. it's part of the API that would allow you to force idempotency in a transaction.  proooobably i could just take it out.

Reply:
in fact it's not even accessible right now. i'm getting rid of it.